pkg/apim/APIMUtility.go

```diff
@@ -0,0 +1,44 @@
+package apim
+
+import (
+	"fmt"
+
+	"github.com/SAP/jenkins-library/pkg/cpi"
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/xsuaa"
+	"github.com/pkg/errors"
+)
+
+//APIMCommonUtils for apim
+type APIMUtils interface {
+	NewAPIM() error
+}
+
+//APIMBundle struct
+type APIMBundle struct {
```

Comment:
type name will be used as apim.APIMBundle by other packages, and that stutters; consider calling this Bundle